meilisearch-core/src/update/settings_update.rs

```diff
@@ -138,6 +138,23 @@ pub fn apply_settings_update(
     Ok(())
 }
 
+fn transform_synonyms(synonyms: BTreeMap<String, Vec<String>>) -> BTreeMap<String, Vec<String>> {
+    synonyms
+        .into_iter()
+        .fold(BTreeMap::new(), |mut map, (key, values)| {
```

Comment:
I am not a big fan 💨 of this `fold`, I would highly prefer a for loop for readability but if you think I am wrong, keep it :)

Reply:
I don't mind, I like fold, but I prefer readability